Makefile

```diff
@@ -410,7 +410,7 @@ configure-psp:
 # Usage example:
 # REPLICATION_FACTOR=3 make cassandra-stress small
 #
-ifeq (cassandra-stress,$(firstword $(MAKECMDGOALS)))
+ifeq (stress-cassandra,$(firstword $(MAKECMDGOALS)))
```

Comment:
Ugh, the action here references the tool that is used underneath, which means `cassandra-stress` ...

Reply:
I had change action name to `stress-cassadra` because of folder name which is `cassandra-stress`. 
If you run `make cassandra-stress` without this change, it makes the operation with folder and not with makefile action